extension/index.googleFunction.js

```diff
@@ -20,12 +31,10 @@ exports.extensionTrigger = async (request, response) => {
       })
     }
   } catch (err) {
-    const errorMessage = `Unexpected error: ${JSON.stringify(
-      serializeError(err)
```

Comment:
Remove the require part for serializeError in the first line.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/630/commits/ba3a6b0ca39bab097f093e6a96d06635d135e8a0

Reply:
It is required now for logging purpose